part-1-ajax/test.js

```diff
@@ -1,76 +1,58 @@
-function getCountryName(code) {
-    $.ajax({
-      //uses the three letter code to get the country's full name
-      }
-    })
-  }
 
-  function parseData(country) {
-    //takes the country data
-    //extracts the name, capital, population, and neighboring countries
-    //iterates through the neighbors array to send off second AJAX request
-  }
+$('form').submit(function(event) {
```

Comment:
you want to clear the information on the page everything you click submit again.